app/migrations/Version20230522141144.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\PreUpAssertionMigration;
+
+final class Version20230522141144 extends PreUpAssertionMigration
+{
+    protected function preUpAssertions(): void
+    {
+        $this->skipAssertion(function (Schema $schema) {
+            return $schema->hasTable("{$this->prefix}plugin_citrix_events");
+        }, sprintf('Table %s already exists', "{$this->prefix}plugin_citrix_events"));
```

Comment:
This is inverted, right? We want to skip the migration if the table does not exist. And the output message should say the opposite too.

Reply:
a clear sign to make coffee!

Reply:
Fixed in efc0a7e8f972d8b772aa2e32d34477e2eaad76f5